### ISSUE TITLE
Use webpack for bundling

### ISSUE THREAD

**Issue by christian-bromann** [baruchvlz/resq#16]
The current bundled version is not useful as it still requires some babel packages. I got a much smaller bundle that assigns to the window object by using webpack and the following config:

```js
const path = require('path')

module.exports = {
    mode: 'production',
    entry: './index.js',
    output: {
        path: path.resolve(__dirname, 'dist'),
        filename: 'resq.bundle.js',
        library: ['window', 'resq'],
    },
}
```

To be able to use it with WebdriverIO it would be great there would be a bundle that assigns itself to the window object. With that I can inject it using the [`executeScript`](https://w3c.github.io/webdriver/#execute-script) command and run queries from the window object, e.g. https://github.com/webdriverio/webdriverio/pull/3797/files#diff-d58e668c0402c2fa779bbf9c90c308a3R5

**Comment by baruchvlz** [comment]
I will separate these 2 (webpack and injecting) into two separate PRs.

**Comment by christian-bromann** [comment]
No PR needed for injecting, it is part of WebdriverIO to work with the bundle that you publish at the end.

**Comment by baruchvlz** [comment]
Published in[ v1.2.0](https://github.com/baruchvlz/resq/tree/v1.2.0).

Note that the class names and function names for the library are being mangled by webpack. This isn't trivial
